packages/webdriverio/src/utils/interception/devtools.js

```diff
@@ -43,6 +42,7 @@ export default class DevtoolsInterception extends Interception {
                     { requestId }
                 ).catch(() => ({}))
 
+                mock.responseHeaders = { ...responseHeaders }
```

Comment:
@christian-bromann is it ok to add `responseHeaders` to `mock` object?

Reply:
makes sense to me